mimic/test/test_nova.py

```diff
@@ -62,6 +62,7 @@ def create_server(helper, name=None, imageRef=None, flavorRef=None,
     if body is None:
         data = {
             "name": name if name is not None else 'test_server',
+            "key_name": key_name if key_name is not None else 'test_key',
```

Comment:
the two mentioned test cases were added below, plan to merge if the build succeeds.